packages/uniforms-bridge-zod/src/ZodBridge.ts

```diff
@@ -28,6 +29,23 @@ function isNativeEnumValue(value: unknown) {
   return typeof value !== 'string';
 }
 
+function getLabel(value: unknown) {
+  return upperFirst(lowerCase(joinName(null, value).slice(-1)[0]));
```

Comment:
You can combine `upperFirst` and `lowerCase` with [`lodash.capitalize`](https://lodash.com/docs/4.17.15#capitalize)
```suggestion
  return capitalize(joinName(null, value).slice(-1)[0]);
```

Reply:
Good catch. I will also apply it to the `uniforms-bridge-json-schema`.

Reply:
Ah, let's leave it then.